src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ file(GLOB_RECURSE HEADERS
         ${CMAKE_CURRENT_SOURCE_DIR}/src/*.h
         ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp)
 
+# build avx2.cpp with AVX2 support
+ie_avx2_optimization_flags(avx2_flags)
+set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/avx2.cpp PROPERTIES COMPILE_OPTIONS "${avx2_flags}")
```

Comment:
```flags are not effectively added. please do following:
set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/src/avx2.cpp PROPERTIES COMPILE_OPTIONS "${avx2_flags}")
```

Reply:
This was the problem with compilation on Linux, thank you! I didn't realize the path to avx2.cpp was wrong.